File: C/1_hello_world.c
#include <stdio.h>
#include <mpi.h>

int main(argc, argv)
int argc; char **argv; // or char *argv[];
{
    int rank, size;
    MPI_Init(&argc, &argv);
    MPI_Comm_rank(MPI_COMM_WORLD, &rank);
    MPI_Comm_size(MPI_COMM_WORLD, &size);

    printf("Hello world! I'm %d of %d\n", rank, size);

    MPI_Finalize();
    return 0;
}

File: Fortran/1_hello_world.f90
program MPI_Ex
    implicit none
    include '/home/anirbankopty/Softwares/MPICH/mpich-install/include/mpif.h'

    INTEGER :: rank, size, ierror
    call MPI_Init(ierror)
    call MPI_Comm_rank(MPI_COMM_WORLD, rank, ierror)
    call MPI_Comm_size(MPI_COMM_WORLD, size, ierror)

    PRINT "(A,I2,A,I2)", "Hello world! I'm ", rank, " of ", size

    call MPI_Finalize()

end program MPI_Ex